### ISSUE TITLE
support direct access to classes

### ISSUE THREAD

**Issue by chelm** [DigitalGlobe/gbdxtools#91]
HI all - Im starting to push code up that's exploring an alternative path for accessing classes in gbdxtools. The goal will be to support using classes without having to first init an `Interface`.

Still working out details but I think we can support the same exact features that the primary interface gives us (eg auth / sessions) but allow classes like `Catalog`  and `Task` et al to init auth interfaces using a singleton instance pattern. 

So in short we'll be able to do things like this

```
from gbdxtools import S3, Task, Workflow, Ordering 
order =  Ordering.order(...)
task = Task(...)
```

**Comment by nricklin** [comment]
this one is done via previously merged PR
